client_delivery/shared/src/commonMain/kotlin/data/remote/model/BaseResponse.kt

```diff
@@ -0,0 +1,24 @@
+package data.remote.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class BaseResponse<T>(
+    @SerialName("value")
+    val value: T?,
+    @SerialName("isSuccess")
+    val isSuccess: Boolean = true,
+    @SerialName("status")
+    val status: ResponseStatus
+)
+
+@Serializable
+data class ResponseStatus(
+    @SerialName("errorMessages")
+    val errorMessages: Map<String, String>? = null,
```

Comment:
u don't need to make him by default value

Reply:
thanks for your review, we alrady did this fix at another branch "D